tools/packages/minion/minion.init

```diff
@@ -21,15 +21,44 @@ DESC="Minion"
 PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
 MINION_HOME="@INSTPREFIX@"
 SYSCONFDIR="@SYSCONFDIR@"
+STOP_TIMEOUT=10
```

Comment:
Seems like this variable isn't named correctly, since it's the number of retries. Maybe STOP_RETRIES instead?